website/docs/ecosystem/azure_cosmos_db.md

```diff
@@ -5,7 +5,9 @@
 
 [Azure Cosmos DB](https://learn.microsoft.com/en-us/azure/cosmos-db/introduction) is a fully managed [NoSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/distributed-nosql), [relational](https://learn.microsoft.com/en-us/azure/cosmos-db/distributed-relational), and [vector database](https://learn.microsoft.com/azure/cosmos-db/vector-database). It offers single-digit millisecond response times, automatic and instant scalability, along with guaranteed speed at any scale. Your business continuity is assured with up to 99.999% availability backed by [SLA](https://azure.microsoft.com/support/legal/sla/cosmos-db).
 
-Azure Cosmos DB simplifies and expedites your application development by being the single database for your operational data needs, from caching to backup to vector search. Within AutoGen, it services as the unitary database service for [caching](../topics/llm-caching.md), logging, RAG, and other workflows. Many other organizations rely on it as the data infrastructure for a [variety of scenarios](https://learn.microsoft.com/en-us/azure/cosmos-db/use-cases) like AI, digital commerce, Internet of Things, and booking management. Its various [APIs](https://learn.microsoft.com/en-us/azure/cosmos-db/choose-api) allow you to keep all your operational data models, including relational, document, vector, key-value, graph, and table, in one database service.
+Your can simplify your application development by using this single database service for all your operational data needs, from [geo-replicated distributed cache](https://medium.com/@marcodesanctis2/using-azure-cosmos-db-as-your-persistent-geo-replicated-distributed-cache-b381ad80f8a0) to backup to [vector database](https://learn.microsoft.com/en-us/azure/cosmos-db/vector-database). You can see how this is done within AutoGen.
+
+Many other organizations rely on Azure Cosmos DB as the data infrastructure for [a variety of scenarios](https://learn.microsoft.com/en-us/azure/cosmos-db/use-cases), including retail and marketing, IoT and telematics, gaming, social, and personalization. Its various [APIs](https://learn.microsoft.com/en-us/azure/cosmos-db/choose-api) allow you to keep all your operational data models, including relational, document, vector, key-value, graph, and table, in one database service.
```

Comment:
Can you add a list of links to AutoGen documentation and notebooks related to CosmosDB integration? e.g., the cache.